### ISSUE TITLE
搜索重定向问题

### ISSUE THREAD

**Issue by Goojoe** [leptonai/search_with_lepton#52]
之前可以通过https://search.lepton.run/search?q=%s 但是现在无论输入都会被重定向到https://search.lepton.run/search?rid=2Sy3-GbxuujtwD__WSRTS

**Comment by Yangqing** [comment]
Yeah we changed the interface a little bit - you can still use https://search.lepton.run/search?q=%s to access the service, but after the answer is generated, it will redirect to the second url so you can copy and share it - it is cached in KV.

**Comment by Goojoe** [comment]
不不，我的意思是所有的搜索内容都回重定向到那个页面，而不是给出我的搜索结果
